apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html

```diff
@@ -26,24 +26,48 @@
                     <div class="due">
                         <span class="mr-b1 due-amount">{{localeData?.customer_due}}</span>
                         <span class="total-amount pull-right vendor-due-text">
-                            <span>{{amountSettings.baseCurrencySymbol}}</span> {{ totalRecievable | giddhCurrency
-                            }}</span>
+                            <!-- <span>{{amountSettings.baseCurrencySymbol}}</span>
+                            {{ totalRecievable | giddhCurrency }} -->
+                            <span class="d-inline-flex">
+                                <amount-field [amount]="totalRecievable" [currencySymbol]="amountSettings.baseCurrencySymbol"
+                                    [currencyCode]="activeCompany.baseCurrency">
+                                </amount-field>
+                            </span>
+                        </span>
                     </div>
                     <div class="clearfix">
                         <small>{{localeData?.pending_invoices}}</small>
-                        <small class="pull-right">{{ ReceivableDurationAmt | giddhCurrency }}</small>
+                        <!-- <small class="pull-right">{{ ReceivableDurationAmt | giddhCurrency }}</small> -->
+                        <small class="d-inline-flex pull-right">
+                            <amount-field [amount]="ReceivableDurationAmt" [currencySymbol]="false"
```

Comment:
Do check currencySymbol is not provided but currencyCode is provided. Check this for Gulf country company